packages/edit-navigation/src/components/header/index.js

```diff
@@ -56,6 +56,9 @@ export default function Header( {
 				</h2>
 
 				<DropdownMenu
+					contentClassName={
+						'edit-navigation-header__component-popover'
+					}
```

Comment:
nitpick, here and in a few other places it could be shortened to this:
```suggestion
					contentClassName="edit-navigation-header__component-popover"
```